src/main/resources/graphics/scenery/volumes/vdi/AmanatidesJumps.comp

```diff
@@ -0,0 +1,1360 @@
+#version 450
```

Comment:
This needs cleanup and profiling, also the name should be changed to something that reflects what this shader is actually doing 🙃

Reply:
I have removed some commented code and renamed the file to `RaycastVDI.comp`. The code has previously been profiled and there is not much scope for further optimization as a vast majority of the frame time is spent waiting on essential memory accesses. At least until we have SSBOs in scenery. Some minor optimizations may be possible but they would require working through the code and algorithm in detail. I could work on this in the future but would prefer for the PR to be merged in the meantime @skalarproduktraum